sdk/core/azure-core/azure/core/pipeline/policies/distributed_tracing.py

```diff
@@ -25,47 +25,48 @@
 # --------------------------------------------------------------------------
 """Traces network calls using the implementation library from the settings."""
 
-from azure.core.pipeline import PipelineRequest, PipelineResponse
 from azure.core.tracing.context import tracing_context
-from azure.core.tracing.abstract_span import AbstractSpan
 from azure.core.tracing.common import set_span_contexts
 from azure.core.pipeline.policies import SansIOHTTPPolicy
 from azure.core.settings import settings
 
 try:
-    from urllib.parse import urlparse
+    from urlparse import urlparse  # type: ignore
 except ImportError:
-    from urlparse import urlparse
+    from urllib.parse import urlparse  # type: ignore
```

Comment:
It seems as though `six` is an acceptable dependency, could use `six.moves` for this instead of handling manually

Reply:
done